ResourcesCard.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useEffect } from "react";
+
+import { sendEvt } from "services/event.service";
+
+import imgFallback from "img/resource_fallback.png";
+
+import styles from "./ResourcesCard.module.scss";
+
+const ResourcesCard = () => {
+  const [resources, setResources] = useState([]);
+  const [primaryResource, setPrimaryResource] = useState(none);
+  const [secondaryResources, setSecondaryResources] = useState([]);
+
+  var sendEventTracking = (resourceUuid) => {
+    const evt = new Object();
+    evt = {
+      event_name: "web_dashboard_card_" + r.id + "_" + r.title + "_tap"
+    };
+    sendEvt(evt);
+    console.log("sending event");
+  };
+
+  // TODO
+  useEffect(async () => {
+    let req = await getResources(userId);
+    setResources(req.resources);
+
+    let primaryResource = resources[0];
+    let secondaryResources = resources.slice(1, 5);
+    setPrimaryResource(primaryResource);
+    setSecondaryResources(secondaryResources);
+    return;
```

Comment:
We may want to return a cleanup function to abort the getResources call to avoid memory leaks and duplicate calls for setting state.

```
useEffect(() => {
    let isCancelled = false;

    let {resources} = await getResources(userId);

   if (!isCancelled) {
    setResources(resources);
  }

  return () => {
    isCancelled = true;
  };
}, []);